syncd/scripts/poesyncdmgrd

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import os
+import signal
+import subprocess
```

Comment:
Why this script is needed who will be calling it ?

Reply:
This is meant to be called by the supervisord process instead of spawning syncd (poesyncd) directly. The main purpose of the script is to check if PoE is supported on the platform and if we should be running syncd at all.